lib/opensusebasetest.pm

```diff
@@ -341,6 +341,7 @@ sub wait_boot {
             }
             wait_serial('GNU GRUB') || diag 'Could not find GRUB screen, continuing nevertheless, trying to boot';
             select_console('svirt');
+            save_svirt_pty;
```

Comment:
do we understand why this call is necessary here? `save_svirt_pty` should have been called in the call to `activate_console('svirt')` so probably implicitly just above or in line 336. Why is necessary there then?